projects/data-atlas-app/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
```

Comment:
Can you please explain what this means and why we need it

Reply:
I can't tell you unfortunately, it was generated automatically. We could try out different modifications if you like?